doc/source/user/problem_method.rst

```diff
@@ -635,5 +638,78 @@ parameters
        the minimum across all 3 dimensions), at the highest refinement
        level.`
 
-   
+
+``"accretion"``: accretion
+==============================
+
+For cells within a spherical accretion zone around a sink particle, mass is removed
+(i.e., the values of the density field are reduced) and added to the sink particle.
+The momentum change of gas is in the accretion zone due to the mass loss is accounted
+for by changing the momentum of the sink particle, so that total momentum is
+conserved. The amount of mass removed is determined by which "flavor" of accretion is
+chosen (specified by the ``"accretion:flavor"`` parameter), as well as the values
+of the "density threshold" (specified by ``"accretion:density_threshold"``) and the
+"maximum mass fraction" (specified by ``"accretion:max_mass_fraction"``).
+
+In ``"threshold"`` flavor accretion, the change in density of each cell zero if the current
+density is below the density threshold. If the current density is above the density threshold,
+the change in density is the current density minus the density threshold, or the maximum mass
+fraction times the current density, whichever is smaller.
+
+In ``"bondi_hoyle"`` flavor accretion, the density change in each cell is calculated according
+to the method described in Mark R. Krumholz et al 2004, ApJ, 611, 399. Furthermore, the
+density change is limited in the same way as in ``"threshold"`` accretion.
+
+In ``"flux"`` flavour accretion, the density change in each cell is calculated according to the
+method described in Andreas Bleuler & Romain Teyssier 2004, MNRAS, 445, 4015-4036.
+Furthermore, the density change is limited in the same way as in ``"threshold"`` accretion.
+
+This method can only be used if ``"merge_sinks"`` is also used, with ``"merge_sinks"`` preceding
```

Comment:
`merge_sinks` ensures that sink particles cannot have overlapping accretion zones (since if they do overlap, they are merged). This is a fundamental assumption in my implementation of the `accretion` method.